Core/src/TrackFitting/GainMatrixSmoother.cpp

```diff
@@ -35,8 +35,16 @@ Result<void> GainMatrixSmoother::calculate(
                   predictedCovariance(prev_ts).inverse();
 
   if (G.hasNaN()) {
-    // error = KalmanFitterError::SmoothFailed;  // set to error
-    // return false;                             // abort execution
+    // ACTS_ERROR("Gain smoothing matrix G has NaNs");
+
+    // ACTS_ERROR("Filtered covariance:\n" << filteredCovariance(ts));
+    // ACTS_ERROR("Jacobian:\n" << jacobian(prev_ts));
+    // ACTS_ERROR("Predicted covariance:\n" << predictedCovariance(prev_ts));
+    // ACTS_ERROR("Inverse of predicted covariance:\n"
+    //            << predictedCovariance(prev_ts).inverse());
+
+    // ACTS_ERROR("Gain smoothing matrix G:\n" << G);
+
```

Comment:
were these commented changes intended?

Reply:
Yes I think this could be useful for debugging in the future